inference-engine/src/transformations/src/transformations/op_conversions/convert_maxpool_downgrade.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_maxpool_downgrade.hpp"
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <transformations/utils/utils.hpp>
+#include "itt.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(pass::ConvertMaxPool8ToMaxPool1, "ConvertMaxPool8ToMaxPool1", 0);
+
+
+pass::ConvertMaxPool8ToMaxPool1::ConvertMaxPool8ToMaxPool1() {
+    MATCHER_SCOPE(ConvertMaxPool8ToMaxPool1);
+
+    auto maxpool_v8_pattern = pattern::wrap_type<ngraph::opset8::MaxPool>();
+
+    matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        auto maxpool_v8_node = std::dynamic_pointer_cast<ngraph::opset8::MaxPool>(m.get_match_root());
+        if (!maxpool_v8_node)
+            return false;
+
+        if (maxpool_v8_node->get_output_target_inputs(1).size() != 0)
+            return false;
```

Comment:
fuse both if in one

Reply:
I do not clearly understand this comment.
We cannot replace MaxPool-8 with MaxPool-1, if there are consumers on the second output port, MaxPool-1 has only one output port

Reply:
replace it to if(!maxpool_v8_node&& maxpool_v8_node->get_output_target_inputs(1).size() != 0)

Reply:
Maybe to ` if(!maxpool_v8_node || maxpool_v8_node->get_output_target_inputs(1).size() != 0) return false;`, not `&&`?

Reply:
yea, sorry